diff --git a/drush.prompt.sh b/drush.prompt.sh
index fd8fee1..19ccd7c 100644
--- a/drush.prompt.sh
+++ b/drush.prompt.sh
@@ -28,7 +28,7 @@ if [ -n "$(type -t __git_ps1)" ] && [ "$(type -t __git_ps1)" = function ] && [ "
   #   username@hostname ~/working-directory (git-branch)[@drush-alias] $
   #
   # See http://ss64.com/bash/syntax-prompt.html for customization options.
-  export PROMPT_COMMAND='__git_ps1 "\u@\h \w" "$(__drush_ps1 "[%s]") \\\$ "'
+  export PROMPT_COMMAND='__git_ps1 "\[\033[01;32m\]\u@\h \[\033[01;34m\]\w\[\033[00m\]" "$(__drush_ps1 "[%s]") \[\033[01;34m\]\\\$\[\033[00m\] "'
 
   # PROMPT_COMMAND is used in the example above rather than PS1 because neither
   # Git nor Drush color hints are compatible with PS1. If you don't want color
